bundles/mapping/toolbar/default-buttons.js

```diff
@@ -21,6 +21,26 @@ Oskari.clazz.category(
             // Without a conf, all buttons are included
             return true;
         },
+        /**
+         * @method _getLinkUuid
+         * @private
+         * 
+         * @return {String} current view uuid, if public view. Otherwise returns uuid of system default view with same srs as current view.
+         */
+        _getLinkUuid: function () {
+            if(Oskari.app.isPublic()) {
+                return Oskari.app.getUuid();
+            }
+            // not public -> get the a system appsetup with matching srs
+            var srs = Oskari.getSandbox().getMap().getSrsName();
+            var matchingSystemAppsetup = Oskari.app.getSystemDefaultViews().find(function(appsetup) {
```

Comment:
If I interpret this correctly IE doesn't support Array.find() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find